### DISCUSSION TITLE
https://takeshima.page/

### DISCUSSION THREAD

**Discussion by giscus** [sdk-xyz/blog-comments#12] [General]
# https://takeshima.page/

https://takeshima.page/

**Comment by sdk-kr** [comment]
![3669208aa2c020](https://github.com/sdk-xyz/blog-comments/assets/16316626/b5344ccb-086d-402e-9f0c-6bb35b1c3f9a)
![1c776eb24a8c16](https://github.com/sdk-xyz/blog-comments/assets/16316626/b3b3a1af-212c-423f-ad25-f03cedf8e4fd)


 "日本が作成した地図にも、独島（日本の古い地名としての松島）は韓国の土地とされています。"
(English): "Even on maps created by Japan, Dokdo (formerly known in Japanese as Matsushima) is recognized as Korean territory."
Korea : 일본이 만든 지도에도 독도(일본 고 지명 송도)는 한국땅이라고 되어 있어요

**Comment by christlkh** [comment]
Le nom originel de Takeshima est Dokdo. Dokdo est un territoire souverain de la République de Corée.
